project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.1
+sbt.version=1.5.4
```

Comment:
Big bump.

Reply:
Rather than doing this here, which impacts not only the whole build, but also potentially *users* of the sbt plugin, you can put this in ` sbt-crossproject-test/src/sbt-test/new-api/shared-cross-sources/project/build.properties`, so that it only applies to running that specific test, and not to the rest of the build.

Reply:
It's also possible that this is what causes the other test to fail. So moving that to the relevant test may fix the CI.